pkgs/applications/networking/instant-messengers/ripcord/default.nix

```
@@ -0,0 +1,32 @@
+{ stdenv, fetchurl, appimageTools }:
+
+let
+  pname = "Ripcord";
+  version = "0.4.18";
+  name = "${pname}-${version}";
+
+  src = fetchurl {
+    url = "https://cancel.fm/dl/${pname}-${version}-x86_64.AppImage";
```

```suggestion
    url = "https://cancel.fm/dl/${name}-x86_64.AppImage";
```
Since you've defined name anyway